index.bs

```diff
@@ -3562,7 +3562,7 @@ extension=] that does not otherwise require any result values MUST return a valu
 An extension defines one or two request arguments. The <dfn>client extension input</dfn>,
 which is a value that can be encoded in JSON, is passed from the [=[RP]=] to the client
 in the {{CredentialsContainer/get()}} or {{CredentialsContainer/create()}} call,
-while the [=CBOR=] <dfn>authenticator extension input</dfn> is
+while the [=CBOR=] [=authenticator extension input=] is
```

Comment:
I disagree with removing the dfn here and moving it to the processing steps.  It makes more sense here.  Please revert this change.

Reply:
Ok.

Reply:
Ok. The reason I did this was so that the `<dfn>` would be in the same section as the corresponding one for the authenticator outputs.